ti/instrumentation/utils.go

```diff
@@ -219,18 +219,18 @@ func downloadFile(ctx context.Context, path, url string, fs filesystem.FileSyste
 	// Create the file
 	out, err := fs.Create(path)
 	if err != nil {
-		return fmt.Errorf("could not create directory: %s", err)
+		return err
```

Comment:
It was difficult from the current logging to determine which of the step in this function failed, so I added this additional logging improvement, I think we should keep them to make error diagnosis easier.